### PR TITLE
FEM-2738 Added support for automaticallyWaitsToMinimizeStalling (from iOS 10)

### PR DESCRIPTION
**Pull request by x-NR-x**
### Description of the Changes

Added an option to change the player's automaticallyWaitsToMinimizeStalling.

### CODE REVIEW

File: Classes/Player/PKPlayerSettings.swift [hunk 1]
```diff
@@ -41,10 +41,25 @@ typealias SettingsChange = ((PlayerSettingsType) -> Void)
...
+    ///
+    /// You may want to set this property to NO when you need precise control over playback start times, e.g., when synchronizing multiple instances of AVPlayer. If the value of this property is NO, reasonForWaitingToPlay cannot assume a value of AVPlayerWaitingToMinimizeStallsReason. This implies that setting rate to a non-zero value in AVPlayerTimeControlStatusPaused will cause playback to start immediately as long as the playback buffer is not empty. When the playback buffer becomes empty during AVPlayerTimeControlStatusPlaying and playback stalls, playback state will switch to AVPlayerTimeControlStatusPaused and the rate will become 0.0.
+    ///
+    /// @available(iOS 10.0, *) via AVPlayer
+    @objc public var automaticallyWaitsToMinimizeStalling = true {
+        didSet {
+            print("Nilit did set \(automaticallyWaitsToMinimizeStalling)")
```
Comment (noamtamim) [inline Classes/Player/PKPlayerSettings.swift]: Remove :-)

File: Classes/Player/PKPlayerSettings.swift [hunk 2]
```diff
@@ -41,10 +41,25 @@ typealias SettingsChange = ((PlayerSettingsType) -> Void)
...
+    ///
+    /// You may want to set this property to NO when you need precise control over playback start times, e.g., when synchronizing multiple instances of AVPlayer. If the value of this property is NO, reasonForWaitingToPlay cannot assume a value of AVPlayerWaitingToMinimizeStallsReason. This implies that setting rate to a non-zero value in AVPlayerTimeControlStatusPaused will cause playback to start immediately as long as the playback buffer is not empty. When the playback buffer becomes empty during AVPlayerTimeControlStatusPlaying and playback stalls, playback state will switch to AVPlayerTimeControlStatusPaused and the rate will become 0.0.
+    ///
+    /// @available(iOS 10.0, *) via AVPlayer
```
Comment (noamtamim) [inline Classes/Player/PKPlayerSettings.swift]: Too long, and we don't expose all of the properties etc that are mentioned.
I think we should just write one short sentence on what it does and link to Apple's doc for more info.

**Review by noamtamim — CHANGES_REQUESTED** [review]

File: Classes/Player/PKPlayerSettings.swift (hunk 1 above)
Comment (x-NR-x) [reply Classes/Player/PKPlayerSettings.swift]: ops lol

File: Classes/Player/PKPlayerSettings.swift (hunk 2 above)
Comment (x-NR-x) [reply Classes/Player/PKPlayerSettings.swift]: Wanted to do that at the beginning, but then thought to mention only the two states.
Ok, will change.

**Review by noamtamim — APPROVED** [review]

**Pull request merged**